crates/bevy_input/src/mouse.rs

```diff
@@ -26,6 +26,13 @@ pub struct MouseMotion {
     pub delta: Vec2,
 }
 
+/// A mouse scroll wheel event, where x represents horizontal scroll and y represents vertical scroll.
+#[derive(Debug, Clone)]
+pub struct MouseWheel {
+    pub x: f64,
```

Comment:
Should this be f32? Most operations are going to be f32, and the event itself only exposes f32. The precision is going to be the same